kv.go

```diff
@@ -980,3 +965,55 @@ func (kv *kvs) Status() (KeyValueStatus, error) {
 
 	return &KeyValueBucketStatus{nfo: nfo, bucket: kv.name}, nil
 }
+
+// KeyValueStoreNames is used to retrieve a list of key value store names
+func (js *js) KeyValueStoreNames() <-chan string {
+	ch := make(chan string)
+	l := &streamLister{js: js}
+	l.js.opts.streamListSubject = fmt.Sprintf(kvSubjectsTmpl, "*")
```

Comment:
Do you know if this subject will work with the JS Prefix? Or whether it needs something like `js.apiSubj()` to make it so?

Reply:
This is the filter subject for `STREAM.LIST` api and points to `$KV.%s.>`. AFAIK, `CreateKeyValue()` creates a stream with interest in this subject (`%s` is the name), so there is no need for `$JS` prefix. Or do I misunderstand it?

Reply:
ahh right, ok then there is no need for the prefix there.